ibm_catalog.json

```diff
@@ -169,10 +357,27 @@
               ]
             },
             {
-              "key": "en_key_ring_name"
+              "key": "provider_visibility",
```

Comment:
should be hidden.

`"hidden": true`

Reply:
Not sure yet if we are going to hide this - it will be discussed in todays deep dive

Reply:
As per the deep dive, please set `"hidden": true`